cluster/configs/src/main/java/com/linkedin/openhouse/cluster/configs/ClusterPropertiesUtil.java

```diff
@@ -0,0 +1,33 @@
+package com.linkedin.openhouse.cluster.configs;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.util.Optional;
+import org.springframework.util.ClassUtils;
+import org.springframework.web.servlet.HandlerInterceptor;
+
+public class ClusterPropertiesUtil {
```

Comment:
this util is moved from tablesMVC configurer, to a generic and testable location.